insights/parsers/nmcli.py

```diff
@@ -191,3 +192,39 @@ def parse_content(self, content):
     def disconnected_connection(self):
         """(list): It will return all the disconnected static route connections."""
         return self._disconnected_connection
+
+
+add_filter(Specs.nmcli_conn_show_uuids, ["connection.uuid", "connection.id"])
```

Comment:
Is this required to add in the parser? If so, the spec will be 100% collected. It would be good to add them in the rules, so, when such rules were retire, we do not need to collect this spec in that case.

Reply:
@wushiqinlou - removed filter